class Validation_Exception(Exception):
    pass